e2e/capacity/transactions.test.ts

```diff
@@ -181,7 +181,8 @@ describe('Capacity Transactions', function () {
 
           const { eventMap } = await call.payWithCapacity();
           assertEvent(eventMap, 'capacity.CapacityWithdrawn');
-          assertEvent(eventMap, 'messages.MessagesInBlock');
+          // messages.MessagesInBlock in block might not be on this transaction if there are others
```

Comment:
Hit this with the other activity on testnet

Reply:
The `eventMap` is only the map of events for that transaction, not the block. So it doesn't always exist in the map. I could look for it in the block, but that doesn't really tell us much more than just the success.